src/includes/getting-started-primer/native.mdx

```diff
@@ -2,6 +2,10 @@ The Sentry Native SDK is intended for C and C++. However, since it builds as a d
 
 Sentry also offers higher-level SDKs for platforms with built-in support for native crashes:
 
-- [_Android_](/platforms/android/)
-- [_Apple (Cocoa)_](/platforms/apple/)
-- [_Electron_](/platforms/electron/)
+- <PlatformLink platform="android" icon="true" />
+- <PlatformLink platform="apple" icon="true" />
+- <PlatformLink platform="electron" icon="true" />
+
+Using a framework? Take a look at our specific guides to get started.
```

Comment:
I may add this to the common content, based on your feedback. If so, I'll remove it here as it will become redundant. Thanks for the suggestion.

Reply:
Cool, thanks!

Reply:
Based on your feedback and some internal convo, I added this to the common content (turning it off for platforms that don't have related guides), so you won't need it here.

Reply:
Awesome! Thanks :)